Microsoft.Toolkit.Uwp.SampleApp/Shell.xaml.cs

```diff
@@ -155,6 +156,29 @@ public void RegisterNewCommand(string name, RoutedEventHandler action)
             while (innerTextbox == null);
         }
 
+        public async Task RefreshXamlRenderAsync()
+        {
+            if (_currentSample != null)
+            {
+                var code = string.Empty;
+                if (_currentSample.PropertyDescriptor != null &&
+                    _currentSample.PropertyDescriptor.Options.Count > 0 &&
+                    InfoAreaPivot.SelectedIndex == 0)
```

Comment:
Use the SelectedItem syntax like we use elsewhere?

Reply:
Good point, fixed